README.md

```diff
@@ -64,23 +64,32 @@ This project can be run in a [Docker](https://www.docker.com/) container.
 You need a local database instance listening on port 8080. Here is how to set that up:
 
 ``` sh
-docker run --name mysql -e MYSQL_ROOT_PASSWORD=password -p 3306:3306 -d mysql
-export LEDGER_DB_URI=mysql://root:password@localhost/fivebells
-npm run migrate
+docker run --name db -e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres
```

Comment:
Would prefer a name that's more descriptive and less likely to collide with a container the user already has, e.g. `--name five-bells-ledger-db`.